modules/nodes-scheduler-taints-tolerations-about.adoc

```diff
@@ -216,9 +244,23 @@ These tolerations ensure that the default pod behavior is to remain bound for fi
 
 You can configure these tolerations as needed. For example, if you have an application with a lot of local state, you might want to keep the pods bound to node for a longer time in the event of network partition, allowing for the partition to recover and avoiding pod eviction.
 
-Daemon set pods are created with NoExecute tolerations for the following taints with no tolerationSeconds:
+DaemonSet pods are created with NoExecute tolerations for the following taints with no tolerationSeconds:
```

Comment:
This is interesting, because a daemon set spawns pods on all the nodes. Still, I think _daemon set pods_ is more correct than `DaemonSet` pods.

Is this something specific that happens only to pods spawns by a daemon set? Maybe

> If a pod is spawned by a daemon set ...

But I lack enough context here to appreciate what's being described.

Oh, and

`NoExecute`

needs back ticks.

So does `tolerationSeconds`